edx/edx-ora2
# Hide step actions for response if in studio

**TL;DR -** 
Added a has_real_user prop based on whether the runtime is bound to a real user.

This is now linked such that nothing below the response shows up if it is true.

JIRA: [AU-58](https://openedx.atlassian.net/browse/AU-58)

**Developer Checklist**


**Testing Instructions**

View an ORA in studio and verify the Save, file upload, and actions (submit button) don't show up
Verify that they do for the LMS view.

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:


FYI: @edx/masters-devs-gta
